diracx-routers/src/diracx/routers/version.py

```diff
@@ -0,0 +1 @@
+DIRACX_MIN_CLIENT_VERSION = "0.0.1"
```

Comment:
I'd prefer to have that constant in `routers/__init__.py`.  `routers.version` is too misleading as a name I believe